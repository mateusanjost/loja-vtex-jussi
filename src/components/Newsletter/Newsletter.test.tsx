import TestRenderer from "react-test-renderer";
import NewsletterComponent from "./Newsletter";

describe("Newsletter", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<NewsletterComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
