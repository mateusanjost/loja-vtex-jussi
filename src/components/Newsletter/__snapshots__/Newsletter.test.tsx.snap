// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newsletter should match snapshot 1`] = `
<div
  className="sc-hKFxyN sc-eCApnc okUlG hNAVpI"
>
  <div
    className="sc-hKFxyN sc-fujyAs okUlG jebvdG"
  >
    <div
      className="sc-gtsrHT sc-jSFjdj lejSiq bChOrd"
    >
      receba novidades da nossa Ã¡rea de produtos digitais.
    </div>
    <div
      className="sc-gtsrHT sc-jSFjdj lejSiq bChOrd"
    >
       
      <form
        className="sc-pNWdM hEEKuF"
      >
        <input
          className="sc-kEqXSa BdmAb"
          type="text"
          value="Digite seu e-mail"
        />
        <button
          className="sc-jrsJWt bYVrbb"
          disabled={true}
        >
          CADASTRAR
        </button>
      </form>
    </div>
  </div>
</div>
`;
